modules/programs/qutebrowser.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+let
```

Comment:
I would suggest adding a `with lib;` here to avoid having to use `lib.` all over.

Reply:
It's not my personal preference, but I understand the importance of a consistent codebase. Done.